.vscode/launch.json

```diff
@@ -184,28 +184,28 @@
             "stopAll": true
         },
         {
-            "name": "Debug in Office (Edge)",
+            "name": "Debug in the Microsoft 365 app (Edge)",
             "configurations": [
-                "Attach to Frontend in Office (Edge)"
+                "Attach to Frontend in the Microsoft 365 app (Edge)"
             ],
             "preLaunchTask": "Start Teams App Locally & Install App",
```

Comment:
- Update all "Start Teams App Locally & Install App" to "Start Teams App Locally". 
- Delete "Start Teams App Locally & Install App" and "Install app in Teams" tasks from `tasks.json`.